application/app/src/main/java/uk/nhs/digital/uec/api/util/PostcodeFormatterUtil.java

```diff
@@ -26,6 +25,10 @@ public  static String formatPostcode(String postcode) {
     outCode = str.substring(0, outEnd);
     inCode = str.substring(outEnd);
 
+    System.out.println("Outcode length: "+outCode.length());
```

Comment:
Could you please use log instead of sysout statements

Reply:
Thanks, good catch, I was trying to reverse engineer what this class did and left this in by accident.  Thank you!